boards/xtensa/mt8195_adsp/Kconfig.board

```diff
@@ -0,0 +1,5 @@
+# Copyright 2023 The ChromiumOS Authors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
One question here: you will still use the loading mechanism from Linux with SOF zephyr on MTK? Not sure why this loader is necessary.